docs/UserGuide.md

```diff
@@ -192,7 +180,6 @@ Action | Format, Examples
 **add** | `add n/NAME p/PHONE a/ADDRESS d/DAY t/TIME [tag/TAG]` <br> e.g., `add n/Jenny Talia p/91234567 a/Yishun Avenue 4 d/FRIDAY t/1800 tag/female`
 **list** | `list`
 **edit** | `edit INDEX [n/NAME] [p/PHONE] [a/ADDRESS] [d/DAY] [t/TIME] [tag/TAG]` <br> e.g., `edit 8 a/Changi Airport d/SATURDAY`
-**search** | `search CONSTRAINT`<br> e.g., `search female or search available`
 **delete** | `delete INDEX`<br> e.g.,`delete 3`
-**drive** | `drive INDEX INDEX INDEX...`<br> e.g., `drive 2 3 4`
+**drive** | `drive n/DRIVER_NAME p/DRIVER_PHONE c/INDEX [c/INDEX c/INDEX...]`<br> e.g., `drive n/Ben p/91234567 2 3 4`
```

Comment:
Just like the one above, the index numbers in the example are missing the c/ prefix